src/main/java/uk/gov/digital/ho/hocs/aws/SQSConfiguration.java

```diff
@@ -19,7 +19,7 @@
     @Bean(name = "sqsClient")
     public AmazonSQS sqsClient(@Value("${case.creator.sqs.access-key}") String accessKey,
```

Comment:
The environment variable set in kube is CASE_CREATOR_SQS_ACCESS_KEY, which equate to case.creator.sqs.access.key
```suggestion
    public AmazonSQS sqsClient(@Value("${case.creator.sqs.access.key}") String accessKey,
```

Reply:
Just had a look at the docs and it seems both ways are supported. All good.